Makefile

```diff
@@ -211,74 +202,6 @@ delete-tag:
 docker-otelcol:
 	COMPONENT=otelcol $(MAKE) docker-component
```

Comment:
The target `docker-component` can probably be removed as well, as `docker-otelcol` is the only one calling it.

Reply:
Removed